### PR TITLE
Adding Griduniverse test

### PR DESCRIPTION
**Pull request by vlall**
In response to #106. Run a test to see that Griduniverse runs with chrome in headless mode, using a bot. And that the bot is capable of scoring points during gameplay.

### CODE REVIEW

File: dlgr/griduniverse/experiment.py [hunk 1]
```diff
@@ -1318,11 +1318,24 @@ def game_loop(self):
...
 
-    def analyze(self, data):
-        return self.average_score(data)
+    def player_feedback(self, data):
+        df = data.infos.df
+        engagement = int(json.loads(df.questions.list[-1][-1])['engagement'])
+        difficulty = int(json.loads(df.questions.list[-1][-1])['difficulty'])
+        return engagement, difficulty
+
+    def average_pay_off(self, data):
```
Comment (suchow) [inline dlgr/griduniverse/experiment.py]: `average_pay_off` => `average_payoff`

File: dlgr/griduniverse/demo.py [hunk 2]
```diff
@@ -11,6 +11,6 @@
     num_dynos_worker=participants,
 )
 
-results = experiment.analyze(data)
+results = experiment.average_score(data)
```
Comment (suchow) [inline dlgr/griduniverse/demo.py]: Can you keep the `results = experiment.analyze(data)` API and instead have the `analyze` method return a JSON structure with results, e.g.:

```
{
    "average_payoff": 0.03,
    "average_score": 1.5,
}
```

The benefit of doing it this way is that it scales well with additional analyses and maintains a single API for all experiments that want to specify analyses. To replicate an experiment, you'd then run the following, regardless of the experiment:

```
import dallinger

experiment = dallinger.experiments.MyExperiment()
data = experiment.run()
results = experiment.analyze(data)
```

File: dlgr/griduniverse/experiment.py [hunk 3]
```diff
@@ -1318,11 +1318,24 @@ def game_loop(self):
         self.publish({'type': 'stop'})
         return
 
-    def analyze(self, data):
-        return self.average_score(data)
+    def player_feedback(self, data):
+        df = data.infos.df
+        engagement = int(json.loads(df.questions.list[-1][-1])['engagement'])
```
Comment (suchow) [inline dlgr/griduniverse/experiment.py]: I think you can get the infos table in other formats directly, e.g., `data.infos.list` or `data.infos.json`. See dallinger/data.py.

Comment (vlall) [inline dlgr/griduniverse/experiment.py]: After looking at data.py, I'm not exactly sure how getting the infos table as a list is preferable to getting the infos table as a dataframe.

**Pull request merged**